Modelica/Electrical/Spice3.mo

```diff
@@ -10,7 +10,7 @@ package Spice3 "Library for components of the Berkeley SPICE3 simulator"
     annotation (Documentation(info="<html>
 <h4>Overview of Spice3 library</h4>
 <p>The Spice3 library is a Modelica library that contains some models of the Berkeley SPICE3 analog simulator.</p>
-<p><u>General information about the analog simulator SPICE3 </u></p>
+<p><u>General information about the analog simulator SPICE3</u></p>
```

Comment:
Better to use <h5> here instead of <p><u>. Fits also to "The Spice3-library for Modelica" below.

Reply:
@tobolar I did not touch HTML structure. This could go to separate PR.